common/static/common/js/discussion/views/discussion_content_view.js

```diff
@@ -494,7 +494,7 @@
 
             DiscussionContentShowView.prototype.getAuthorDisplay = function() {
                 return _.template($('#post-user-display-template').html())({
-                    username: this.model.get('username') || null,
+                    username: realUserName || null,
```

Comment:
This PR will require changes if `realUserName` variable will be set as the `data-` attribute